app/controllers/api/v1/rents_controller.rb

```diff
@@ -1,15 +1,17 @@
 module Api
   module V1
-    class RentsController < ApplicationController
+    class RentsController < ApiController
+      rescue_from Pundit::NotAuthorizedError, with: :pundit_not_authorized_error
       include Wor::Paginate
 
       def index
-        @rents = Rent.includes(:book, :user).where(user_id: params[:id])
-        render_paginated @rents
+        render_paginate @rents = policy_scope(Rent)
```

Comment:
try not to assign inside a method call:

Use

```
@rents = policy_scope(Rent)
render_paginated @rents

# or

render_paginated(policy_scope(Rent))

```

instead.

Reply:
Change to the second option.